applications/conversations/models/class.conversationmodel.php

```diff
@@ -771,7 +771,7 @@ public function addUserToConversation($ConversationID, $UserID) {
         }
 
         // First define the current users in the conversation
-        $OldContributorData = $this->getRecipients($ConversationID);
+        $OldContributorData = $this->getRecipients($ConversationID, 1000);
```

Comment:
If you're dialing up the artificial recipient ceiling, why not apply it to all calls or update the default in the function? `ConversationModel::getRecipients` is currently called in four places in Vanilla. It seems like a bad idea to have it continue to return 20, everywhere else.